lib/zv.ml

```diff
@@ -8,23 +8,23 @@ module S = struct
   }
   [@@deriving show, sexp, yojson]
 
-  let neg x = Z.neg x
+  let neg = Z.neg
```

Comment:
Quick question, do these eta reduce in ocaml?

Reply:
Yes they do eta reduce but having explicit parameter helps with type inference, so it's better to write it out in func signatures

Reply:
Ah I see, I've reverted to only the README.md changes